core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -87,6 +87,9 @@ angular.module('oppia').directive('storyNodeEditor', [
             $scope.nodeIdToTitleMap =
               $scope.story.getStoryContents().getNodeIdsToTitleMap(
                 $scope.storyNodeIds);
+            $scope.nodeIdToDescriptionMap =
```

Comment:
Why do you need this map at all? For title, I think we needed it to mark destination node titles, but we don't need  that for descriptions.

Reply:
I have removed it. Done.